Generative art blogpost (#2)
aivillage/aiv_website: _posts/articles/2022-08-16-genart.md

# AI Village Art Day @ DEF CON 30

> Today, models that appear to generate novel outputs conditioned on text are becoming the mainstay of popular culture.

Reviewer: This is definitely an exaggeration. Maybe we do this?

```suggestion
Models that generate interesting outputs conditioned on text are now a permanent piece of popular culture.
```

> New groups of users are starting to adopt generative architectures and bring new modes (experiential, creative) of interacting with large models.

Reviewer: What are you trying to say here- can you give more detail (examples of groups, what architectures exist, what are these modes of interaction). Probably this should be expanded to a full paragraph

> The understanding of model performance is becoming increasingly qualitative and subjective, as is evident in the latest benchmarks for text-to-image models.

Reviewer: Are you trying to say that non-technical users are now forming opinions on models and model performance, or are you trying to say that ML researchers disagree on what metrics should be used for benchmarking? Either way, more detail would be helpful!

Author: yeah, great point of clarification -- I would basically aim at showing that there is now a plurality of ways of engaging with the models, and some (those practiced by artists / those posting to Twitter) are more experiential; in ways, it is helpful to researchers (who may have a different modality in which to appreciate all the new models) to understand how the model is being used out there _and_ perhaps suggest new models for incentivizing exploration of unintended / notable behavior that is reliant on humans, rather than more principled exploration (but somehow, hopefully, can suggest novel approaches)

if nothing else - "it looks pretty" and "inductive bias over many iterations" crowds talking to each other could be entertaining, if nothing else (and hopefully no theology this time around)

> We would like to give a platform to the artists and those building large foundation models to come together, and begin to form a shared understanding of each other's approaches.

Reviewer: ```suggestion
At DEF CON 30, AI Village will host both artists and those building large foundation models, in order to facilitate a shared understanding of each other's approaches.
```

> ## Context

Reviewer: The below should be in paragraph or bulleted form, as it is this is messy and unpolished/unfinished

generative art 
## Art Day
### Ongoing exhibits
### Workshops

> - interactieve Google Colab session

Reviewer: ```suggestion
- Interactive Google Colab session
```

### Meet & Greet

> ## Call for artists!

Reviewer: Do we have a link?
Shouldn't we also have a Call for Foundational Model builders?

Reviewer 2: Too complicated.

> ## References

Reviewer: ```suggestion
If you're interested in art generation, the below links are a great starting point.
```

- https://openai.com/blog/clip/
- https://openai.com/dall-e-2/
- https://imagen.research.google/

> - [On the Opportunities and Risks of Foundation Models](https://arxiv.org/abs/2108.07258)

Reviewer: Why is this link the only one with context?

Author: At its core, acrasia! But, more meat now 🥩 hope it's fresh enough

> ## Try it yourself!

Reviewer: ```suggestion
```

> If you wanted to get a headstart, experience generative models and play with their implementation!

Reviewer: ```suggestion
```
